README.md

```diff
@@ -190,7 +190,7 @@ If you don't need your ACME client to have a specific domain name (`mydomain.tes
 
 An upside of this approach is that any pod in Kubernetes will be able to find its to the actual web-server using the domain name, and not only those like Pebble using the configurable DNS server.
 
-A downside of this approach is that requests must go to this specific domain name as it is the only reference to the Kubernetes service. This can be resolved in two ways.
+A downside of this approach is that requests must go to this specific domain name as it is the only reference to the Kubernetes service. This can be resolved
```

Comment:
is this missing an end?

Reply:
Yepp! I still have some work in the README.md left since the removal of pebble-challtestsrv!